cli/src/main/java/com/box/l10n/mojito/cli/command/DropViewCommand.java

```diff
@@ -0,0 +1,172 @@
+
+    @Parameter(names = {"--csv"}, required = false, description = "When showing drops, output info in csv format so that it is easily parsable by scripts.")
+    Boolean showCsvOutput = false;
+
+    @Parameter(names = {"--json"}, required = false, description = "When showing drops, output info in json format so that it is easily parsable by scripts.")
+    Boolean showJsonOutput = false;
+
+    @Parameter(names = {"--dropid", "-i"}, required = false, arity = 1, description = "Show information only about the drop with the given id")
+    Long dropid = -1L;
+
+    @Autowired
+    CommandHelper commandHelper;
+
+    @Autowired
+    Console console;
+
+    @Autowired
+    DropClient dropClient;
+
+    private void outputInfo(Drop drop) {
+        if (dropid != -1 && drop.getId().longValue() != dropid.longValue()) return;
+        if (showCsvOutput) {
+            consoleWriter.a(drop.getId()).a(",").a(drop.getName());
+
+            if (Boolean.TRUE.equals(drop.getCanceled())) {
+                consoleWriter.a(",CANCELED");
+            } else if (drop.getLastImportedDate() == null) {
+                consoleWriter.a(",NEW");
+            } else {
+                consoleWriter.a(",").a(drop.getLastImportedDate());
+            }
+        } else {
+            consoleWriter.
+                    a("id: ").fg(Color.MAGENTA).a(drop.getId()).reset().
+                    a(", name: ").fg(Color.MAGENTA).a(drop.getName()).reset();
+
+            if (Boolean.TRUE.equals(drop.getCanceled())) {
+                consoleWriter.fg(Color.GREEN).a(" CANCELED");
+            } else if (drop.getLastImportedDate() == null) {
+                consoleWriter.fg(Color.GREEN).a(" NEW");
+            } else {
+                consoleWriter.a(", last import: ").fg(Color.MAGENTA).a(drop.getLastImportedDate());
+            }
+        }
+        consoleWriter.println();
+    }
+
+    @Override
+    public void execute() throws CommandException {
+
+        Repository repository = commandHelper.findRepositoryByName(repositoryParam);
+
+        Map<Long, Drop> numberedAvailableDrops = getNumberedAvailableDrops(repository.getId());
+
+        if (numberedAvailableDrops.isEmpty()) {
+            consoleWriter.newLine().a("No drop available").println();
+        } else {
+            if (!showCsvOutput && !showJsonOutput) consoleWriter.newLine().a("Drops available").println();
+
+            logger.debug("Display drops information");
+            if (showJsonOutput) {
+                try {
+                    ObjectMapper objectMapper = new ObjectMapper();
+                    String value = objectMapper.writeValueAsString(
+                        (dropid == -1) ?
+                            numberedAvailableDrops :
+                            numberedAvailableDrops.get(dropid)
+                    );
+                    consoleWriter.a(value).println();
+                } catch (JsonProcessingException e) {
+                    System.out.println(e);
```

Comment:
I think serialization error shouldn't be caught and printed here - instead execute() should just throw it so that the command would fail with a non-zero exit code

Reply:
now throws a new CommandException with the json exception as the cause